presto-hive/src/main/java/io/prestosql/plugin/hive/s3/HiveS3Config.java

```diff
@@ -457,4 +459,30 @@ public HiveS3Config setRequesterPaysEnabled(boolean requesterPaysEnabled)
         this.requesterPaysEnabled = requesterPaysEnabled;
         return this;
     }
+
+    public String getS3ProxyHost()
+    {
+        return s3ProxyHost;
+    }
+
+    @Config("hive.s3.proxy-host")
```

Comment:
Let's change this to `hive.s3.proxy.address` and make it take `HostAndPort`. See `HdfsConfig.setSocksProxy()` for an example.